.github/workflows/ci.yml

```
@@ -113,7 +113,11 @@ jobs:
         run: composer audit
 
       - name: Run test suite
-        run: composer run-tests-with-clover
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2044:warning:1:13: For loops over find output are fragile. Use find -exec or a while read loop

(shellcheck)